deploy_heroku.py

```diff
@@ -143,7 +143,7 @@ def main(heroku_app: str, heroku_api_key: str, git_commit_hash: str, json_event_
         raise RuntimeError("Heroku release command failed! See Heroku release logs for detailed information.")
 
     print(f"New release version is {latest_heroku_release.version}")
-    print(f"::set-output name=release_version::{latest_heroku_release.version}")
+    subprocess.run(f"echo 'release_version={latest_heroku_release.version}' >> $GITHUB_OUTPUT", check=True, shell=True)
```

Comment:
Alternative to your solution can be:

```suggestion
    with open(os.environ["GITHUB_OUTPUT"], "a") as fh:
        print(f"release_version={latest_heroku_release.version}", file=fh)
```

Reply:
Thanks for the feedback. I don't have a strong opinion on this, and I'm also ok with removing the version output if we no longer needed.

Reply:
Okay I searched on GitHub at it seems that no public repository is using it, but we don't know if someone uses it in a private repo:
https://github.com/search?q=%22moneymeets%2Faction-heroku-deploy%22&type=code

I think we can remove it. If there is somebody using it, we will be notified.

Please remove it also from https://github.com/moneymeets/action-heroku-deploy/blob/master/action.yml#L14.

The `id` in some of our repositories can be removed as well (e.g.) https://github.com/moneymeets/moneymeets-backoffice/blob/1882976a0c119b3b5f68144412cbe4e4216724d2/.github/workflows/deploy.yml#L20, but this can be done independently.

Reply:
I have removed the release version output. I will check our Heroku deployment workflows and will open pull requests to remove the `id`, where it is present.